specification/hybridkubernetes/resource-manager/Microsoft.Kubernetes/preview/2021-03-01-preview/connectedClusters.json

```diff
@@ -0,0 +1,801 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ConnectedKubernetesClient",
+    "version": "2021-03-01-preview",
```

Comment:
>2021-03-01-preview [](start = 16, length = 18)

Would it make sense to keep the preview API version as a different month then ga? I know technically we dont need to do this.. Just trying to see if this would cause confusion? If ARM API board is fine then we dont need to.

Reply:
Don't think it matters much but incrementing date for GA would be appreciated in case people are trying to alphabetically order the versions and confuse themselves.